targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/TARGET_DISCO_L475VG_IOT01A/PeripheralPins.c

```diff
@@ -335,3 +335,22 @@ const PinMap PinMap_CAN_TD[] = {
     {PD_1,  CAN_1, STM_PIN_DATA(STM_MODE_AF_PP, GPIO_NOPULL, GPIO_AF9_CAN1)}, // PMOD 4
     {NC,    NC,    0}
 };
+
+const PinMap PinMap_QSPI_DATA[] = {
```

Comment:
Hi
You should rebase, MBED_WEAK is missing ?

Reply:
will check and do

Reply:
rebased and fixed